### ISSUE TITLE
Parallel buildx builds stop with build errors - a single build works fine

### ISSUE THREAD

**Issue by clauspruefer** [docker/buildx#2152]
### Contributing guidelines

- [X] I've read the [contributing guidelines](https://github.com/docker/buildx/blob/master/.github/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When running multiple docker buildx jobs on Ubuntu 22.04.3 (Jammy Jellyfish) in multiple bash windows, build breaks with error messages.

### Expected behaviour

Parallel builds of different dockerfiles should work in parallel without raising error messages.

### Actual behaviour

Parallel builds randomly fail with different error messages raised.

### Buildx version

v.0.11.2 9872040

### Docker info

```text
Client:
 Version:    24.0.5
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx

Server:
 Containers: 6
  Running: 2
  Paused: 0
  Stopped: 4
 Images: 17
 Server Version: 24.0.5
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 
 runc version: 
 init version: 
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.2.0-33-generic
 Operating System: Ubuntu 22.04.3 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 12
 Total Memory: 23.44GiB
 Name: dev-vm
 ID: 2b9444da-67f5-4e57-9c6e-120e23f1d15b
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

### Builders list

```text
No special information.
```

## Dockerfile #1

```docker
FROM ubuntu:latest
MAINTAINER Claus Prüfer

ARG DEBIAN_FRONTEND=noninteractive

ARG DB_DEB_FILE=x0-db_0.98rc_all.deb

COPY ./x0/docker/tmp/apt-sources.list /etc/apt/sources.list
COPY ./x0/docker/tmp/environment-db.sh ./environment.sh

COPY ./x0/docker/scripts/start-postgresql.sh /root/start-postgresql.sh
COPY ./$DB_DEB_FILE ./

RUN rm /var/lock
RUN mkdir -p /var/lock/

RUN apt-get -qq update -y

RUN TZ="Europe/Berlin" apt-get -qq install -y tzdata locales

RUN apt-get -qq install -y ./$DB_DEB_FILE

CMD /root/start-postgresql.sh

EXPOSE 5432
```

## Dockerfile #2
```
FROM ubuntu:latest
MAINTAINER Claus Prüfer

ARG DEBIAN_FRONTEND=noninteractive

ARG APP_DEB_FILE=x0-app_0.98rc_all.deb
ARG APP_PSYCOP_DEB_FILE=x0-psycopg2-patched_0.98rc_all.deb

COPY ./x0/docker/tmp/apt-sources.list /etc/apt/sources.list
COPY ./x0/docker/tmp/environment-app.sh ./environment.sh

COPY ./$APP_DEB_FILE ./
COPY ./$APP_PSYCOP_DEB_FILE ./

RUN rm /var/lock
RUN mkdir -p /var/lock/

RUN apt-get -qq update -y

RUN TZ="Europe/Berlin" apt-get -qq install -y tzdata locales

RUN apt-get -qq install -y ./$APP_DEB_FILE
RUN apt-get -qq install -y ./$APP_PSYCOP_DEB_FILE

RUN /var/lib/x0/app-setup/bin/fs-permissions.sh

RUN chown www-data:www-data /var/www

CMD /var/lib/x0/sys/docker-start-apache.sh

EXPOSE 80
EXPOSE 443

```

**Comment by crazy-max** [comment]
> fail with different error messages raised.

Can you post your logs please?

> Builders list

Builders list would be necessary as well.

Thanks

**Comment by clauspruefer** [comment]
> > fail with different error messages raised.
> 
> Can you post your logs please?
> 
> > Builders list
> 
> Builders list would be necessary as well.
> 
> Thanks

Following @weekend.

**Comment by clauspruefer** [comment]
> > fail with different error messages raised.
> 
> Can you post your logs please?
> 
> > Builders list
> 
> Builders list would be necessary as well.
> 
> Thanks

If i build with disabled cache and log-output `--no-cache --progress=plain ... &> build.log` for all images (tested with 4 parallel builds) everything works fine. I propose to investigate further with build-cache enabled.

Furthermore i reinstalled Win11 where my Linux Dev-VM (including docker.io and buildx) is running on, VM is reimported, so i suppose it indeed is the cache. What to do? Close / Reopen Issue on demand? My time is limited currently.

**Comment by clauspruefer** [comment]
Please be patient, logs will follow.

**Comment by crazy-max** [comment]
No logs or `buildx ls` output followed the "logs will follow" in December 2023, and the VM was reinstalled afterwards, so there is nothing to act on.

Closing as stale. If this still happens on a current buildx and Docker Engine, please open a new issue with `docker buildx version`, `docker buildx ls` and the failing build output.
